js/board.js

```diff
@@ -140,4 +135,4 @@ export default {
             deck.updateCardPos($card, newPos);
         }
     }
-}
```

Comment:
nit: what changed here? can you revert this no-op line?

Reply:
I can't figure out how - I didn't add/remove this myself as far as I know